core/templates/components/forms/schema-based-editors/schema-based-list-editor.component.html

```diff
@@ -48,12 +51,11 @@
       Please ensure that the list has no duplicates.
     </div>
 
-    <!-- The margin-left is added here to left-align the button with the previous list items. -->
     <div class="oppia-new-tab-button-container">
-      <button [style.opacity]="(!isOneLineInput || isAddItemButtonPresent || (localValue && localValue.length && localValue[localValue.length - 1].length)) ? 1 : 0"
+      <button
+              class="{{ (!isOneLineInput || isAddItemButtonPresent || (localValue && localValue.length && localValue[localValue.length - 1].length)) ? 'btn btn-secondary btn-sm ml-0 e2e-test-add-list-entry' : 'oppia-btn-no-display' }}"
```

Comment:
```suggestion
              class="{{ (!isOneLineInput || isAddItemButtonPresent || (localValue && localValue.length && localValue[localValue.length - 1].length)) ? 'btn btn-secondary btn-sm ml-0 e2e-test-add-list-entry' : 'd-none' }}"
```
Can you check if directly using the bootstrap class causes no issues?